test/data/assetcreator/chaincode/smartcontract.go

```diff
@@ -24,3 +24,11 @@ func (s *SmartContract) CreateAsset(ctx contractapi.TransactionContextInterface,
 	ctx.GetStub().SetEvent("AssetCreated", assetJSON)
 	return ctx.GetStub().PutState(name, assetJSON)
 }
+
+func (s *SmartContract) GetAsset(ctx contractapi.TransactionContextInterface, name string) (string, error) {
```

Comment:
No objections to this being added to the chaincode... but was it intended to be used in a test or something? Seems unused at this point.

Reply:
Ah this is being used in my other branch for additional tests :) It shouldn't have been in this commit. I'll remove it now for clarity.